app/styles/_tooltip.less

```diff
@@ -2,6 +2,10 @@
 // Tooltips
 // --------------------------------------------------
 
+.tooltip-default-icon {
+  font-size: @font-size-base - 1;
+  vertical-align: top;
```

Comment:
I like creating a common style, but currently most other tooltip icons aren't top-aligned. For instance,

![openshift web console 2017-10-31 16-59-39](https://user-images.githubusercontent.com/1167259/32248735-fc0055e0-be5c-11e7-8c4a-feb665091103.png)

Reply:
Yeah, thought about that. Maybe add a todo to sweep through and common class.